compiler/src/diagnostics/output.re

```diff
@@ -39,9 +40,11 @@ let exn_to_lsp_error = (exn: exn): option(lsp_error) => {
   };
 };
 
+// kept lenses as an empty array so older LSPs don't crash
```

Comment:
I'm not super concerned about this at this stage since we're still iterating rapidly, but it's good to think ahead 🙂